helm-cider.el

```diff
@@ -462,7 +464,7 @@ The old and new functions are those specified in
 
 ;;;###autoload
 (define-minor-mode helm-cider-mode
-  "Use Helm for CIDER."
+    "Use Helm for CIDER."
```

Comment:
That looks wrong.

Reply:
Another consequence of using `common-lisp-indent-function` it seems...I guess I'll push the docstring up onto the same line as the `define-minor-mode`.